src/content/docs/alerts/admin/rules-limits-alerts.mdx

```diff
@@ -107,6 +107,20 @@ Limits and rules pertaining to New Relic <InlinePopover type="alerts"/>:
       </td>
     </tr>
 
+    <tr>
+      <td>
+        Alerts query scan operations per minute, per account ([learn more](#query-scan-limit))
```

Comment:
`Alerts` -> `Alert` to be consistent with the limit name `Alert query scan operations per minute`

Reply:
pushed a fix